### PR TITLE
Consumed capacity getter methodes excludes INVALID_STATES from respective resource state enum

### PR DESCRIPTION
**Pull request by chrisvdg**

### CODE REVIEW

**Comment by chrisvdg** [comment]
Maybe the `Cloudspace` and `Account` enum classes also need an `INVALID_STATES` item?

**Comment by grimpy** [comment]
I totally agree with your comment I rather have you add the invalid states to cloudspace than that you use the invalidstatus from machine in a cloudspace query

**Comment by chrisvdg** [comment]
@grimpy, thinking more about it, checking Memory and CPU consumed capacity should be using the Machine invalid state, for the disks a new one should be added actually, but that might come with #1535?
I'll add a temporary enum class for disk one till that get's implemented

File: apps/cloudbroker/cloudbrokerlib/resourcestatus.py [hunk 1]
```diff
@@ -31,3 +31,12 @@ class Account(object):
     CONFIRMED = 'CONFIRMED'
     DISABLED = 'DISABLED'
     
+class Disk(object):
+    ASSINGED = 'ASSIGNED'
```
Comment (grimpy) [inline apps/cloudbroker/cloudbrokerlib/resourcestatus.py]: one of these two words has a typo

**Pull request merged**